### PR TITLE
fix(vertical nav): Add primary navigation code example

### PR DESCRIPTION
**Pull request by dabeng**
This PR is targeted to fix issue #286 .

Here's the [demo](https://rawgit.com/dabeng/patternfly-org/dist-masthead/_site/pattern-library/navigation/vertical-navigation/#code) page.

@dgutride @amarie401 Would you like to review this PR whenever you have time ?

### CODE REVIEW

File: source/_includes/code/navigation/vertical-navigation/code.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+          width="100%" height="650px;" scrolling="no" seamless></iframe>
+</div>
+<p><a href="{{ site.baseurl}}/pattern-library/navigation/vertical-navigation/vertical-navigation-masthead.html" target="_blank">View full page example</a></p>
+<div class="row">
+  <div class="col-md-12">
+    <p class="reference-markup"><a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="markup-3" href="#markup-3">Reference Markup</a></p>
+    <div class="collapse in" id="markup-3">
+      <pre class="prettyprint">{% capture markup_include %}
+<html class="layout-pf layout-pf-fixed">
+...
```
Comment (amarie401) [inline source/_includes/code/navigation/vertical-navigation/code.md]: I would suggest removing the `...`

Comment (amarie401) [reply source/_includes/code/navigation/vertical-navigation/code.md]: @dabeng I also noticed there's a wide gap between where the `scripts` end and where the `nav` code begins. Is there any way to fix that gap or is that how the code is included in the reference markup?

**Comment by dabeng** [comment]
@LHinson I have rename code example to "Primary Navigation" and you can checkout it out on updated [demo page](https://rawgit.com/dabeng/patternfly-org/dist-masthead/_site/pattern-library/navigation/vertical-navigation/#code). As for damaged  PF icon/image, it will work well when released on formal domain instead of rawgit.com.

@amarie401  I have removed symbol ". . ." from source code section. However, are you sure you want to remove all the blank lines on the patternfly.org site. FYI, [Whitespace control of Liquid](https://shopify.github.io/liquid/basics/whitespace/)

**Comment by amarie401** [comment]
@LHinson if all else looks good to you then we can merge this.

**Comment by LHinson** [comment]
Thanks @dabeng!

**Pull request merged**